### PR TITLE
Adds new darkspawn ability - Devour lifeforce 

### PR DESCRIPTION
**Pull request by Notamaniac**
# Document the changes in your pull request

Adds Devour Lifeforce - A ability that instantly kills anyone who has had their will devoured recently and has no mindshield implant- and heals you slightly. Victims who haven't had their will devoured or a mindshield implant won't instantly be killed, but will still take damage (and heal you for a lower amount). They'll also be confused too.

Darkspawn struggle in combat against multiple people, and I'm hoping that giving them a good AOE ability can help them better deal with multiple foes.

# Spriting
devour_lifeforce - Copy

# Wiki Documentation

Devour Lifeforce - Kills any victims within a 7 tile range who has had their will devoured and no mindshield implant. Heals you for all damage. Victims who haven't had their will devoured or a mindshield implant won't instantly be killed, but will still take damage (and heal you for a lower amount). They will aso be confused too.

# Changelog
:cl:  
rscadd: Adds Devour Lifeforce - A new darkspawn ability that instantly kills anyone who has had their will devoured and no mindshield implant - as well as healing you. Victims who haven't had their will devoured or a mindshield implant won't instantly be killed, but will still take (less) damage and be confused (and heal you for a lower amount).
/:cl:

### CODE REVIEW

File: yogstation/code/modules/antagonists/darkspawn/darkspawn_abilities/devour_lifeforce.dm [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	psi_cost = 60 //since this is only useful when cast directly after a succ it should be pretty expensive
+	lucidity_price = 2
+
+/datum/action/innate/darkspawn/devour_lifeforce/Activate()
+	var/mob/living/carbon/human/H = owner
+	if(!do_after(owner, 1 SECONDS, owner))
+		return
+	owner.visible_message(span_boldwarning("[owner] lets out a chilling cry!"), "<span class='velvet bold'>...wjz oanra</span><br>\
+	[span_notice("You devour the life force from everyone nearby.")]")
+	playsound(owner, 'yogstation/sound/ambience/antag/veil_mind_scream.ogg', 100)
```
Comment (tattax) [inline yogstation/code/modules/antagonists/darkspawn/darkspawn_abilities/devour_lifeforce.dm]: you sure 100 is not too loud?

**Comment by ReddicusDragon** [comment]
this is INCREDIBLY FUCKING STUPID, anything that insta KILLS IS FUCKING STUPID

**Comment by Marmio64** [comment]
seems unnecessary while at the same time probably too weak to justify getting. Most people who are going to be well armed enough to warrant using this on them rather than just tendril beating them to death will have a mindshield anyway, and you'd generally want to save up and start veiling people rather than just killing them.

File: yogstation/code/modules/antagonists/darkspawn/darkspawn_abilities/devour_lifeforce.dm (hunk 1 above)
Comment (Notamaniac) [reply yogstation/code/modules/antagonists/darkspawn/darkspawn_abilities/devour_lifeforce.dm]: it's the same volume as the veil mind screm- it didn't seem ear breaking.

**Comment by Marmio64** [comment]
In addition, I forgot this but veil mind was also changed recently so it costs the same Psi and lucidity as this ability except instead of just killing the guy you get a minion, so there really isn't a point to using this over veil in the majority of cases.

**Comment by Notamaniac** [comment]
> In addition, I forgot this but veil mind was also changed recently so it costs the same Psi and lucidity as this ability except instead of just killing the guy you get a minion, so there really isn't a point to using this over veil in the majority of cases.

You're right. I've dramatically increased the range to 7 (as it's a channel), increased the healing and made it still deal damage to people who haven't had their will devoured (as well as heal). I also made it deal clone damage, so it can have a better offensive edge.

Darkspawn lack decent AOE abilities, I'm hoping this gives them a bit more of an offensive edge in terms of engaging multiple enemies.

> this is INCREDIBLY FUCKING STUPID, anything that insta KILLS IS FUCKING STUPID

This has to be as powerful as it is to keep up with veil mind.  It still doesn't remotely match it's power, as noted before.

**Comment by Moltijoe** [comment]
Why does it instantly kill your own veils, this is kinda backwards

**Comment by Notamaniac** [comment]
> Why does it instantly kill your own veils, this is kinda backwards

Should avoid harming people in your faction now

**Comment by Moltijoe** [comment]
instakills are still wack, but at least you aren't shooting yourself in the foot by killing your allies

**Comment by Notamaniac** [comment]
> instakills are still wack, but at least you aren't shooting yourself in the foot by killing your allies

Keep in mind this is essentially what veil mind does on paper - instantly kills the person you use it on and replaces it with a version who is loyal to you.

**Comment by Moltijoe** [comment]
> Keep in mind this is essentially what veil mind does on paper - instantly kills the person you use it on and replaces it with a version who is loyal to you.

you forget there's a flesh and blood player controlling that character. it's why getting killed by a zombie is different than being killed otherwise.

**Comment by Sniblet** [comment]
Death is meaningless in Yogs and there's no way a darkspawn has the time or interest to keep someone dead after doing this to them so it looks like a rare case of an OK instakill

**Comment by ynot01** [comment]
Balance council near unanimously hates this idea (Ling didn't say anything in the thread)

This is a completely unneeded "buff" and would feel extremely shitty to play against as the person being killed.

**Pull request closed without merging**